frontend/src/pages/modelRegistry/screens/utils.ts

```diff
@@ -133,7 +146,13 @@ export const filterModelVersions = (
         return (
           mv.name.toLowerCase().includes(searchLower) ||
           (mv.description && mv.description.toLowerCase().includes(searchLower)) ||
-          getLabels(mv.customProperties).some((label) => label.toLowerCase().includes(searchLower))
+          (mv.customProperties &&
+            getLabels(mv.customProperties).some((label) =>
+              label.toLowerCase().includes(searchLower),
+            )) ||
+          (mv.modelSourceName && mv.modelSourceName.toLowerCase().includes(searchLower)) ||
+          (mv.modelSourceId && mv.modelSourceId.toLowerCase().includes(searchLower)) ||
+          (mv.modelSourceGroup && mv.modelSourceGroup.toLowerCase().includes(searchLower))
```

Comment:
Interesting - it might be a good idea to support searching based on the model source, but since it's only present on the artifact we can't do it here on the version filtering without some additional lookups that will be painful. Best to remove these lines for now.